.pipelines/templates/teardown.yaml

```diff
@@ -0,0 +1,11 @@
+steps:
+  - script: |
+      rm ${LOCAL_SSH_KEY}*
+      az login -t microsoft.com --service-principal -u ${CLIENT_ID} -p ${CLIENT_SECRET}
+      az group delete -n ${AZURE_CLUSTER_NAME} --subscription ${SUBSCRIPTION_ID} --yes --no-wait
```

Comment:
Should we also clean up the e2e image after the job is done?

Reply:
Done!